### DISCUSSION TITLE
Spacy v3 CLI training a blank nlp NER model and retraining it with more data

### DISCUSSION THREAD

**Discussion by noobistz** [explosion/spaCy#7112] [Help: Coding & Implementations] [answered]
Hi all, I have been reading up on spacy's v3.0 documentation but can't seem to find it anywhere. 

When CLI spacy train is used, how do I know if it is training a blank NER model or it is training using a pre-trained NER model like "en_core_web_sm/md/lg"? If i want to do the former, how can I go about doing it?

Also whenever CLI spacy train is used, is it safe to assume that it is training from scratch? How can I choose to train a model that has already been trained with more data?

**Comment by adrianeboyd** [comment, marked as answer]
`spacy train` can do either, it depends on the config.

With a default config from `spacy init config`, it's training a blank model from scratch.

With a custom config, you can use `source` to load in a component from an existing model: https://spacy.io/usage/processing-pipelines#sourced-components

**Reply by noobistz** [reply]
thank you for your help!
